apps/server/src/modules/files/uc/sync-embedded-files.uc.ts

```diff
@@ -27,26 +25,33 @@ export class SyncEmbeddedFilesUc {
 	}
 
 	private async syncEmbeddedFiles(type: AvailableSyncParentType, limit: number) {
-		const [entities, count] = await this.embeddedFilesRepo.findElementsToSyncFiles(type, limit, this.failedIds);
+		const [entities, count] = await this.embeddedFilesRepo.findElementsToSyncFiles(type, limit, []);
```

Comment:
We skip the failedIds logic completly after the change with this.updateEntityLinks(entity, id); right?